docs/setting-up-a-gucdk-project.md

```diff
@@ -18,11 +18,58 @@ To initialise a new project run the following within your repository:
 npx @guardian/cdk@latest new --app [app] --stack [stack] --stage [stage] --package-manager [npm|yarn]
 ```
 
-TIP: Run `npx @guardian/cdk@latest new --help` to find out more about the available flags,
+For example, for the app `riff-raff` we'd do:
+
+```sh
+npx @guardian/cdk@latest new \
+  --app riff-raff \
+  --stack deploy \
+  --stage CODE \
+  --stage PROD
+```
+
+> Tip: Run `npx @guardian/cdk@latest new --help` to find out more about the available flags,
 including `--yaml-template-location` to use when migrating a YAML template to GuCDK.
 
 You should now have an empty stack within `cdk/lib` for you to populate with the necessary resources.
 
+## Defining your infrastructure
+
+### Patterns and Constructs
+
+Now have an empty stack within `cdk/lib`, you can start adding patterns and constructs!
+
+Patterns are TypeScript classes that create the requisite AWS resources for a common architecture. For example:
+
+* [A Play Framework application](https://guardian.github.io/cdk/classes/patterns.GuEc2App.html) running on EC2 instances
+* [A Lambda function](https://guardian.github.io/cdk/classes/patterns.GuScheduledLambda.html) that runs every *n* minutes
+* An API which serves requests using AWS Lambda
```

Comment:
Explaining this distinction was the main motivation for this PR; is it clear enough?